### ISSUE TITLE
Questions on training hyperparameters

### ISSUE THREAD

**Issue by ZuoyunZheng** [Lornatang/CRNN-PyTorch#4]
Hi Lorna,
i was hoping you could shed light on some of the training hyperparmeter choices. I have tried using the ICDAR2013 dataset which is significantly smaller than MJSynth. For training and validation, i use its train and test set respectively.  However the model seems to be converging to outputting complete blanks. I tried making the learning rate to 1e-3 for example and adding a cosine-annealing lr to it. But it doesn't seem to matter. So my questions are (also for when i move onto MJSynth training later):
1. From your experience, where could the problem lie? Is the amount of data from ICDAR2013 too few without data augmentation for this model? 
2. I'm not versed in mixed-precision training, but could this be messing with the model's convergence at the start, since the scaler's scale is somewhere around 1e4?
3. What kind of preprocessing are used for characters that are not supported by the model, i.e. "-"? Do you omit samples containing such characters or do we just ignore it in the ground truth labeling?
5. If you can share, what learning rate schedule did you use for your pretrained model?

Thanks for sharing your experience as you have been! Very appreciated! Thanks in advance!

Regards,
Zheng

**Comment by ZuoyunZheng** [comment]
Just a short update, because I couldn't download the MJSynth dataset you provided, i'm using the original MJSynth dataset for training and IIIT5K for validation. I did very minor tweaks on my [fork](https://github.com/Lornatang/CRNN-PyTorch/compare/main...ZuoyunZheng:CRNN-PyTorch:main) of this repo. All the hyperparameters remains how it was and somehow at epoch 3 the accuracy is still 0.1% or something like that. Could you see maybe where i did something wrong? Thanks in advance!

**Comment by ZuoyunZheng** [comment]
Found the error... Dataset loading the wrong ground truth text...

Thanks for the repo. Was probably the most acccurate implementation on github! Great work!
